index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express')
+const { getAllAuthors, getAuthorsByFilter } = require('./controllers/authors')
+const { getAllGenres, getGenresById } = require('./controllers/genres')
+const { getAllNovels, getNovelsByFilter } = require('./controllers/novels')
+
+
+
+const app = express()
```

Comment:
Watch your inconsistent spacing. Extra new lines just make the code tougher to read as it requires unneeded scrolling. This is the type of thing that will be seen as a potential employer as "junior" or even "careless" neither of which is a good impression.